docs/source/process.rst

```diff
@@ -125,14 +125,19 @@ critical in biological interpretation of ocean sonar data. They influence:
 
 By default, echopype uses the following for calibration:
 
-- EK60 and EK80: Environmental parameters saved with the raw data files.
+- EK60 and EK80: Environmental parameters saved with the raw data files. 
+  For EK60, instrument operators may enter temperature and salinity values into the 
+  `Simrad EK60 software's Environment dialog 
+  <https://www.simrad.online/ek60/ref_english/default.htm?startat=/ek60/ref_english/xxx_para_environment.html>`_
+  and the software will calculate sound speed, or directly enter sound speed instead.
+  Only sound speed is saved into the raw file.
```

Comment:
Could you add "sound absorption" along with sound speed? Both are saved in the data files also.

Reply:
Done